homework-submission/react-hyf/client/src/test.js

```diff
@@ -0,0 +1 @@
+console.log('Hi');
```

Comment:
Hi 👋🏻! Remember to check the files you are committing to the PR (I know you probably glad that it works but for future pull requests).